app/assets/scripts/components/project/sec-panel/class-analytics-chart.js

```diff
@@ -139,34 +133,30 @@ function ClassAnalyticsChart(props) {
         />
       </ChartContainer>
       <Summary>
-        {bounds && (
-          <ClassItem bounds={bounds}>
-            <p> </p>
-            <Prose size='small'>CLASS NAME</Prose>
-            <Prose size='small' className='percent'>
-              AREA KM2
-            </Prose>
-            <Prose size='small' className='percent'>
-              %
-            </Prose>
-          </ClassItem>
-        )}
+        <ClassItem>
```

Comment:
The conditional check for `bounds` (or totalArea, whatever we want to call it) should remain; this ensures that the table header is used only for the class distribution chart and not the class F1 chart. Here's the view in this branch with the current code:
<img width="331" alt="image" src="https://github.com/developmentseed/pearl-frontend/assets/12634024/7e5aecb7-c68d-4fbb-97df-b93fef085317">